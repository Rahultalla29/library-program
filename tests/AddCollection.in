ADD COLLECTION noFile.csv
ADD COLLECTION sample1.csv
ADD COLLECTION sample1.csv
EXIT
