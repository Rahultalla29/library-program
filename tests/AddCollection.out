user: No such collection.

user: 17 books successfully added.

user: No books have been added to the system.

user: Ending Library process.
